src/ducks/settings/AccountsSettings.jsx

```diff
@@ -32,6 +32,7 @@ import { Unpadded } from 'components/Spacing/Padded'
 
 import HarvestBankAccountSettings from './HarvestBankAccountSettings'
 import DisconnectedAccountModal from 'cozy-harvest-lib/dist/components/DisconnectedAccountModal'
+import { utils } from 'cozy-client/dist/models'
```

Comment:
It seems `utils` is exported from `models` which is exported from `cozy-client` — imo it's better to import from the root of the package as it's immune from changes in the file structure.

Reply:
OK, will do in subsequent commits.